packages/sdk/src/controllers/LayoutController.ts

```diff
@@ -76,9 +83,12 @@ export class LayoutController {
      * @param parentId the id of a specific layout, being the parent
      * @returns id of new layout
      */
-    create = async (parentId: Id) => {
+    create = async (parentId: Id, presets?: LayoutPreset[]) => {
         const res = await this.#editorAPI;
-        return res.addLayout(parentId).then((result) => getEditorResponseData<Id>(result));
+
+        return res
+            .addLayout(parentId, presets?.length ? presets : [])
```

Comment:
For backwards compatibility until this is released, the method is now 'addLayouts' in engine (the old addLayout not taking extra string is still there)